engines/python/setup/djl_python/neuron_utils/model_loader.py

```diff
@@ -808,3 +810,119 @@ def partition(self, save_path: str, **kwargs):
         self.load_pipeline(**kwargs)
         self.save_pipeline(save_path)
         return self.pipeline
+
+
+class TNXVllmModelLoader(ModelLoader):
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.generation_config = self.generate_generation_config()
+        self.model_config.generation_config = self.generation_config
+        self.neuron_config = self.generate_neuron_config()
+
+    def generate_generation_config(self):
+        generation_config = NeuronGenerationConfig(
```

Comment:
`do_sample`, `global_top_k`, `per_batch_line`, `deterministic`

for these configs, do the hardcoded values need to be hardcoded, or should we expose them as options?

Reply:
We probably need to. But did not expose them yet, because, this is based on the private drop we got. The values look unstable. Once we have the published artifacts, we could expose them. Or I could add them in as configurable, not document them. I can do this in the next PR.